Measures/AddingInternalLoads/measure.rb

```diff
@@ -23,9 +23,10 @@ def arguments(model)
     args = OpenStudio::Measure::OSArgumentVector.new
 
     # the name of the space to add to the model
+	args << OpenStudio::Measure::OSArgument.makeDoubleArgument("NRF/BGF", true)
     args << OpenStudio::Measure::OSArgument.makeDoubleArgument("ElectricEquipmentPowerPerFloorArea", true)
     args << OpenStudio::Measure::OSArgument.makeDoubleArgument("LightingPowerPerFloorArea", true)
-    args << OpenStudio::Measure::OSArgument.makeDoubleArgument("PeoplePerFloorArea", true)
+    args << OpenStudio::Measure::OSArgument.makeDoubleArgument("FloorAreaPerPerson", true)
```

Comment:
This seems to be the inverse ration of the people per floor area. Just wondering if we considered this change also on the data input side?

Reply:
looks like it down below: